examples/kitchen-sink/page-kit.config.js

```diff
@@ -8,8 +8,16 @@ module.exports = {
   plugins: [
     codeSplitting.plugin(),
     bower.plugin(),
-    sass.plugin({includePaths: [path.resolve('../../bower_components')]}),
-    js.plugin(),
+    sass.plugin({
+      // Sass itself can only attempt to resolve partial files based on the CWD and not relative to
```

Comment:
It's a little pedantic, but could we add `Enabling webpackImporter because Sass itself...` to the beginning of these comments? It's probably obvious if you're familiar with the tools but I was confused the first time I read through this.